File: backend/src/entities/person_skill.go
package entities

import "time"

type PersonSkill struct {
	ID        string `db:"person_skill_id"`
	PersonID  string `db:"person_id"`
	SkillID   string `db:"skill_id"`
	Value     int       `db:"value"`
	Progress  int       `db:"progress"`
	CreatedAt time.Time `db:"created_at"`

	Person    Person
	Skill     Skill     `db:"skill"`
}

File: frontend/lib/entities/personSkill.dart
import 'package:flutter/material.dart';
import 'package:frontend/entities/skill.dart';

class PersonSkill {
  final Skill skill;
  final int value;
  final int progress;

  PersonSkill(
      {@required this.value,
      @required this.progress,
      @required this.skill});

  factory PersonSkill.fromJson(Map<String, dynamic> json) {
    final data = PersonSkill(
      value: json['Value'],
      progress: json['Progress'],
      skill: Skill.fromJson(json['Skill'])

    );
    return data;
  }
}
